modules/react/banner/lib/Banner.tsx

```diff
@@ -94,13 +127,6 @@ export const Banner = createContainer('button')({
     ActionText: BannerActionText,
   },
 })<BannerProps>(({children, ...elemProps}, Element, model) => {
-  const palette = useThemedPalette(model.state.hasError ? 'error' : 'alert');
-  const themedBackgroundStyles: CSSProperties = {
-    '&:hover': {
-      backgroundColor: palette.hover,
-    },
-  };
-
   const isRTL = useIsRTL();
   const borderBottomLeftRadius = isRTL ? 'borderBottomRightRadius' : 'borderBottomLeftRadius';
   const borderTopLeftRadius = isRTL ? 'borderTopRightRadius' : 'borderTopLeftRadius';
```

Comment:
We no longer need these are there are logical CSS properties that can be used:
`border-start-start-radius`
`border-start-end-radius`
`border-end-start-radius`
`border-end-end-radius`

This way RTL is handled by these styles in the browser and not having to handle it manually.

```suggestion
```